File: Java Programming/Competitive Programming/Coding Challenges 8/program4.java
// 4. Accept one number and check whether it is divisible by 5 or not.

import java.util.Scanner;

class Program4
{
    static boolean Check(int iNo)
    {
        if ((iNo % 5) == 0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;
        boolean bRet = false;

        System.out.print("Enter Number: ");
        iValue = sobj.nextInt();

        bRet = Check(iValue);

        if (bRet == true)
        {
            System.out.println("Divisible by 5");
        }
        else
        {
            System.out.println("Not Divisible by 5");
        }

        sobj.close();
    }
}

File: C++ Programming/Competitive Programming/Coding Challenges 5/program4.cpp
// 4. Accept one number and check whether it is divisible by 5 or not.

#include <iostream>
using namespace std;

bool Check(int iNo)
{
    if ((iNo % 5) == 0)
    {
        return true;
    }
    else
    {
        return false;
    }
}

int main()
{
    int iValue = 0;
    bool bRet = false;

    cout << "Enter Number: ";
    cin >> iValue;

    bRet = Check(iValue);

    if (bRet == true)
    {
        cout << "Divisible by 5";
    }
    else
    {
        cout << "Not Divisible by 5";
    }

    return 0;
}